master/buildbot/test/unit/steps/test_source_git.py

```diff
@@ -526,7 +541,11 @@ def test_mode_full_clean_ssh_host_key_1_7(self):
         self.expectOutcome(result=SUCCESS)
         self.expectProperty(
             'got_revision', 'f6ad368298bd941e934a41f3babc827b2aa95a1d', self.sourceName)
-        return self.runStep()
+        yield self.runStep()
+
+        expected = b'#!/bin/sh\nssh -i "%s" -o "UserKnownHostsFile=%s" "$@"\n' \
```

Comment:
Can we use `.format()` for string formatting?

Reply:
Changed to using `.format()`!